.pre-commit-config.yaml

```diff
@@ -1,23 +1,45 @@
 repos:
--   repo: https://github.com/pre-commit/pre-commit-hooks
+  - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v4.1.0
     hooks:
-    -   id: check-yaml
+      - id: check-yaml
         exclude: .github/conda/meta.yaml
-    -   id: end-of-file-fixer
-    -   id: trailing-whitespace
-    -   id: check-case-conflict
-    -   id: check-merge-conflict
--   repo: https://github.com/psf/black
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+      - id: check-case-conflict
+      - id: check-merge-conflict
+  - repo: https://github.com/psf/black
     rev: 22.3.0
     hooks:
-    -   id: black
--   repo: https://gitlab.com/pycqa/flake8
+      - id: black
+  - repo: https://gitlab.com/pycqa/flake8
     rev: 4.0.1
     hooks:
-    -   id: flake8
+      - id: flake8
         types: [file, python]
--   repo: https://github.com/PyCQA/isort
+  - repo: https://github.com/PyCQA/isort
     rev: 5.10.1
     hooks:
-    -   id: isort
+      - id: isort
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.981
+    hooks:
+      - id: mypy
+        # taken from https://github.com/pre-commit/mirrors-mypy/issues/33#issuecomment-735449356
+        args:
+          [
+            "--ignore-missing-imports",
```

Comment:
you can put all the config in `pyproject.toml`, this is what we have in skops:

```
-   repo: https://github.com/pre-commit/mirrors-mypy
    rev: v0.971
    hooks:
    -   id: mypy
        args: [--config-file=pyproject.toml]
        additional_dependencies: [types-requests>=2.28.5]
```

Reply:
Good idea :+1: Makes things more consistent for the `make quality` command and github CI as well. I've made the change.